tests/virtualization/universal/waitfor_guests.pm

```diff
@@ -108,7 +108,6 @@ sub run {
     my $sleep_delay = 1200;
     $sleep_delay = 1800 if (is_xen_host);    # XEN has more guests
     sleep($sleep_delay);    # XXX Get rid of this sleep!
-    start_guests();
```

Comment:
@tonyyuan1 Thanks for the PR. In [ticket][https://progress.opensuse.org/issues/104508#note-6] , you gave reason for removing this is that "virt-install  --wait" can keep virt-install alive. However this start_guests are to "virsh start [guest]" in case they are poweroff after guest installation. So different purpose, we should keep this line,  right?

Reply:
Thanks for review. 
That was explanation quoted from virt-install man. 
If  '--wait' is used the guests are started after their installation. you can see some err like: "error: Domain is already active"  http://openqa.qam.suse.cz/tests/34829#step/waitfor_guests/81

Therefore, It is unnecessary to start again there.

Reply:
I see, then LGTM.